.github/workflows/scraper.yml

```diff
@@ -0,0 +1,72 @@
+name: Postgres Service Example
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
We want the tests to run against the dev branch too.

Reply:
I am just testing this .yml, not sure if it will work at all. I will tell you when it is ready to review.